frontend/src/service/StormGlassService.js

```diff
@@ -1,7 +1,15 @@
 import axios from 'axios';
 
+const setHeaders = () => {
+    return {
+        headers: {
+            "Authorization": "Bearer " + localStorage.getItem("jwtToken")
```

Comment:
Aren't you missing the $-sign here? Mine looks like this " Authorization: `Bearer ${token}`"

Reply:
I think that depends. If you want to use a template literal you need the back-ticks and dollar sign for variables, like you did. Or you can do string concatenation with the plus sign and whatever ticks you prefer.